components/dash-html-components/scripts/data/attributes.json

```diff
@@ -526,6 +526,12 @@
             ],
             "description": "Provides a hint to the user of what can be entered in the field."
         },
+        "playsInline": {
+            "elements": [
+                "video"
+            ],
+            "description": "Sets the video to be played inline / within the element's playback area."
+        },
```

Comment:
This file is actually auto-generated by `scripts/extract-attributes.js` by scraping https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes - and unfortunately `playsinline` is missing there, so when I rerun the build process your edits disappear. We could make a special case for it, but better would be to fix it upstream -> https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#see_also ("Found a problem with this page?")

Reply:
Okay, i made an issue request on their github. Now it will just be a waiting game? 

Again, I am happy with dash-player, so there is no hurry on my end. Just trying to better understand the build for dash components